### PR TITLE
add partial group sopport

### PR DESCRIPTION
**Pull request by redondomarco**
Hello! This is my first contribution to a public project.
add:
-get list of user groups
-get detail of user group
-add member to user group

I hope it is useful!

### CODE REVIEW

File: guacapy/client.py [hunk 1]
```diff
@@ -483,3 +483,47 @@ def add_sharing_profile(self, payload, datasource=None):
...
+        if not datasource:
+            datasource = self.primary_datasource
+        return self.__auth_request(
+            method="GET",
+            url="{}/session/data/{}/userGroups/{}".format(
+                self.REST_API, datasource, usergroup
+            ),
+        )
+
+    def add_member_user_group(self, usergroup, payload, datasource=None):
```
Comment (pschmitt) [inline guacapy/client.py:511]: `edit_group_members` may make more sense here since this method can be used to add or delete group members.

**Comment by pschmitt** [comment]
Thanks for your contribution! I'll merge this once you addressed my comment.

**Comment by redondomarco** [comment]
I already changed the name of the method. It also made sense to me to call it edit.

**Comment by redondomarco** [comment]
Can the same logic be applied for the grant_permission method?

**Comment by pschmitt** [comment]
> Can the same logic be applied for the grant_permission method?

Yeah, it'd make sense to change this is as well. Don't add this to this PR though. Merging in 3.2.1..

**Pull request merged**